docs/todo.rst

```diff
@@ -1,8 +1,47 @@
-.. _todo-label:
+..  _todo-label:
 
-Open issues
+Contributing
 ---------------
 
+If you want to develop for Asterix, you might need packages to test and build the documentation. Run:
+
+.. code-block:: bash
+    
+    $  pip install -e '.[dev,docs]'
+
+To contribute to Asterix, please follow the following steps:
+    1. Make sure your local `master` branch is up-to-date by pulling.
+    2. Create a new branch off `master` with a name of your choice and commit your work.
+    3. When done, open a PR and request a review after ensuring your branch is up-to-date with the base branch you're merging into (usually `master`) and after running the pytests and the linter locally. You can then the run the pytests
+
+    .. code-block:: bash
+
+        $ pytest
+
+    or run the flake8 linter:
+
+    .. code-block:: bash
+
+        $ flake8 . --max-line-length=121 --count --statistics
+
+
+    4. Iterate on the review, once it's approved it will be immediately merged.
+
+**Documentation**
+
+Please update the documentation after modifying Asterix. Docstrings must follow the `numpydoc guidelines <https://numpydoc.readthedocs.io/en/latest/format.html>`_.
```

Comment:
I would be stricter, so instead of saying "Please update the documentation *after* modifying Asterix", I would say: "Please update any affected documentation when working on a PR." That way you won't merge a PR without also having the docs updated. Which is easy to forget, but it's also on the reviewers to remind of that, as you did many times before ^^ (and you should).